core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -691,6 +691,62 @@ public void minute() {
     assertEquals("minute(\"2020-08-17 01:02:03\")", expression.toString());
   }
 
+  private void testMinuteOfHour(FunctionExpression dateExpression, int minute, String testExpr) {
+    assertEquals(INTEGER, dateExpression.type());
+    assertEquals(integerValue(minute), eval(dateExpression));
+    assertEquals(testExpr, dateExpression.toString());
+  }
+
+  @Test
+  public void minuteOfHour() {
+    lenient().when(nullRef.valueOf(env)).thenReturn(nullValue());
+    lenient().when(missingRef.valueOf(env)).thenReturn(missingValue());
+
+    FunctionExpression expression1 = DSL.minute_of_hour(
+        DSL.literal(new ExprTimeValue("01:02:03")));
+    FunctionExpression expression2 = DSL.minute_of_hour(DSL.literal("01:02:03"));
+    FunctionExpression expression3 = DSL.minute_of_hour(
+        DSL.literal(new ExprTimestampValue("2020-08-17 01:02:03")));
+    FunctionExpression expression4 = DSL.minute_of_hour(
+        DSL.literal(new ExprDatetimeValue("2020-08-17 01:02:03")));
+    FunctionExpression expression5 = DSL.minute_of_hour(DSL.literal("2020-08-17 01:02:03"));
+
+
+    assertAll(
+        () -> testMinuteOfHour(expression1, 2, "minute_of_hour(TIME '01:02:03')"),
+        () -> testMinuteOfHour(expression2, 2, "minute_of_hour(\"01:02:03\")"),
+        () -> testMinuteOfHour(expression3, 2, "minute_of_hour(TIMESTAMP '2020-08-17 01:02:03')"),
+        () -> testMinuteOfHour(expression4, 2, "minute_of_hour(DATETIME '2020-08-17 01:02:03')"),
+        () -> testMinuteOfHour(expression5, 2, "minute_of_hour(\"2020-08-17 01:02:03\")")
+    );
+  }
+
+  private void testInvalidMinuteOfHour(String time) {
+    FunctionExpression expression = DSL.minute_of_hour(DSL.literal(new ExprTimeValue(time)));
+    eval(expression);
+  }
+
+  @Test
+  public void minuteOfHourInvalidArguments() {
+    when(nullRef.type()).thenReturn(TIME);
+    when(missingRef.type()).thenReturn(TIME);
+    assertEquals(nullValue(), eval(DSL.minute_of_hour(nullRef)));
```

Comment:
`assertAll`?

Reply:
Fixed in 17510afbbb5ae86cf1df6e209d0726b2bed35735